pkg/reconciler/instances/ory/hydra/syncer.go

```diff
@@ -74,8 +75,7 @@ func restartHydraMaesterDeploymentNeeded(context context.Context, client kuberne
 		return false, err
 	}
 	logger.Debugf("Earliest hydra-maester restart time: %s ", earliestHydraMaesterPodStartTime.String())
-
-	return earliestHydraPodStartTime.After(earliestHydraMaesterPodStartTime), nil
+	return earliestHydraPodStartTime.After(earliestHydraMaesterPodStartTime.Add(startupShift)), nil
```

Comment:
Let's apply this as a temporary solution for syncing Ory, but I'm not generally okay with this.
We should aim for another mechanism checking if the synchronization is needed between ory-hydra and ory-hydra-maester, hard-coded shift time.Duration should be removed in the future.